src/common/Callout.ts

```diff
@@ -0,0 +1,5 @@
+export interface CalloutProps {
+  content: string
+}
```

Comment:
I think we can still add the variant prop first but not use/implement it until later.